webapp-django/crashstats/crashstats/static/crashstats/css/components/table_sorter.less

```diff
@@ -2,7 +2,8 @@
 
 table.tablesorter {
     th {
-        background-color: @light-grey;
+        background-color: @dark-tan;
+        border: solid 1px @grey;
```

Comment:
The tablesorter thing uses border=black to denote which column is being used to sort. Given that, we should make it grey to start with.